scripts/make_random_dataset.py

```diff
@@ -0,0 +1,90 @@
+import glob
+import math
+import os
+import pathlib
+from random import uniform
+from typing import Callable
+
+import gym
+import numpy as np
+import soundfile
+from pynktrombonegym.spaces import ObservationSpaceNames as OSN
+from pynktrombonegym.wrappers import ActionByAcceleration, Log1pMelSpectrogram
+from tqdm.contrib.concurrent import process_map
+
+target_sound_files = glob.glob("data/sample_target_sounds/*.wav")
+sound_seconds_range = (2.0, 3.0)
+data_dir = pathlib.Path(__file__).parent.parent.joinpath("data")
+output_dir = data_dir / "generated_randomly"
+sample_dir = data_dir / "sample_generated_randomly"
+
+if not os.path.exists(output_dir):
+    os.makedirs(output_dir)
+if not os.path.exists(sample_dir):
+    os.makedirs(sample_dir)
```

Comment:
`data_dir` の型は、`<class 'pathlib.PosixPath'>` なので、`os.path` の処理を `pathlib` の処理に変えることができます。
```suggestion
if not output_dir.exists():
    output_dir.mkdir(parents=True)
if not output_dir.exists():
    sample_dir.mkdir(parents=True)
```